qiskit_ibm/managed/managedjob.py

```diff
@@ -120,9 +118,10 @@ def _async_submit(
                         job_tags=job_tags,
                         **run_config)
                 except IBMQBackendJobLimitError:
-                    final_states = [state.value for state in API_JOB_FINAL_STATES]
-                    oldest_running = backend.jobs(limit=1, descending=False,
-                                                  db_filter={"status": {"nin": final_states}})
+                    non_final_states = ["CREATED", "TRANSPILING", "TRANSPILED", "VALIDATING",
+                                        "VALIDATED", "RUNNING", "PENDING_IN_QUEUE", "QUEUED"]
```

Comment:
Could you add this as a API_JOB_NON_FINAL_STATES Tuple in apiconstants file and then use it as a list here? I see it being useful in other places.